src/parser/parser_driver.cpp

```diff
@@ -447,7 +448,7 @@ void ParserDriver::defineGrammar()
 				args[3].getTokenIt()->setValue(std::make_shared<ValueSymbol>(name_text, Expression::Type::Bool));
 				return {};
 			},
-			"tags", "LCB", "metas", "strings", "condition", "RCB", [&](auto&& args) -> Value {
+			"tags", "LCB", "metas", "strings", "variables", "condition", "RCB", [&](auto&& args) -> Value {
```

Comment:
Would it be possible to have 2 possible productions based on whether AvastOnly parsing is enabled or not? Something like this

```c++
auto& rule = _parser.rule("rule");
if (avast_parser_enabled) {
    rule.production(production_with_variables);
} else {
    rule.production(production_without_variables);
}
```

I prefer keeping the grammar as it is if we don't use any additional features.